dymos/examples/ssto/launch_vehicle_ode.py

```diff
@@ -13,10 +14,153 @@ def initialize(self):
+                       units='m/s')
+
+        self.add_input('m',
+                       val=np.zeros(nn),
+                       desc='mass',
+                       units='kg')
+
+        self.add_input('theta',
+                       val=np.zeros(nn),
+                       desc='pitch angle',
+                       units='rad')
+
+        self.add_input('thrust',
+                       val=2100000 * np.ones(nn),
+                       desc='thrust',
+                       units='N')
+
+        self.add_input('Isp',
+                       val=265.2 * np.ones(nn),
+                       desc='specific impulse',
+                       units='s')
+        # Outputs
+        self.add_output('xdot',
+                        val=np.zeros(nn),
+                        desc='velocity component in x',
+                        units='m/s')
+
+        self.add_output('ydot',
+                        val=np.zeros(nn),
+                        desc='velocity component in y',
+                        units='m/s')
+
+        self.add_output('vxdot',
+                        val=np.zeros(nn),
+                        desc='x acceleration magnitude',
+                        units='m/s**2')
+
+        self.add_output('vydot',
+                        val=np.zeros(nn),
+                        desc='y acceleration magnitude',
+                        units='m/s**2')
+
+        self.add_output('mdot',
+                        val=np.zeros(nn),
+                        desc='mass rate of change',
+                        units='kg/s')
+
+        self.add_output('rho',
+                        val=np.zeros(nn),
+                        desc='density',
+                        units='kg/m**3')
+        # Setup partials
+        ar = np.arange(self.options['num_nodes'])
+
+        self.declare_partials(of='rho', wrt='y', rows=ar, cols=ar)
+        self.declare_partials(of='xdot', wrt='vx', rows=ar, cols=ar, val=1.0)
+        self.declare_partials(of='ydot', wrt='vy', rows=ar, cols=ar, val=1.0)
+
+        self.declare_partials(of='vxdot', wrt='vx', rows=ar, cols=ar)
+        self.declare_partials(of='vxdot', wrt='m', rows=ar, cols=ar)
+        self.declare_partials(of='vxdot', wrt='theta', rows=ar, cols=ar)
+        self.declare_partials(of='vxdot', wrt='y', rows=ar, cols=ar)
+        self.declare_partials(of='vxdot', wrt='thrust', rows=ar, cols=ar)
+
+        self.declare_partials(of='vydot', wrt='m', rows=ar, cols=ar)
+        self.declare_partials(of='vydot', wrt='theta', rows=ar, cols=ar)
+        self.declare_partials(of='vydot', wrt='vy', rows=ar, cols=ar)
+        self.declare_partials(of='vydot', wrt='y', rows=ar, cols=ar)
+        self.declare_partials(of='vydot', wrt='thrust', rows=ar, cols=ar)
+
+        self.declare_partials(of='mdot', wrt='thrust', rows=ar, cols=ar)
+        self.declare_partials(of='mdot', wrt='Isp', rows=ar, cols=ar)
+
+        # # Complex-step derivatives
+        # self.declare_partials(of='*', wrt='*', method='cs')
```

Comment:
With this PR (https://github.com/OpenMDAO/OpenMDAO/pull/1862) these lines should work now - can remove the other `declare_partials` calls, and removal of `compute_partials`